ci/docker/install/centos7_python.sh

```diff
@@ -31,5 +31,5 @@ curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
 python2.7 get-pip.py
 python3.6 get-pip.py
 
-pip2 install nose pylint numpy nose-timer requests h5py scipy
-pip3 install nose pylint numpy nose-timer requests h5py scipy
\ No newline at end of file
+pip2 install nose cpplint==1.3.0 pylint==1.8.3 'numpy<1.15.0,>=1.7.2' nose-timer 'requests<2.19.0,>=2.18.4' h5py scipy
```

Comment:
Is this tested?

Reply:
Please pin the versions and dont specify a range